apps/playwright/tests/admin/pool-candidate.spec.ts

```diff
@@ -94,6 +96,208 @@ test.describe("Pool candidates", () => {
+      .click();
+    await appPage.page
+      .getByRole("radio", { name: "This claim has been verified" })
+      .click();
+    await appPage.page.getByRole("spinbutton", { name: "Year" }).fill("2030");
+    await appPage.page
+      .getByRole("combobox", { name: "Month" })
+      .selectOption("01");
+    await appPage.page.getByRole("spinbutton", { name: "Day" }).fill("25");
+    await appPage.page.getByRole("button", { name: "Save changes" }).click();
+    await expect(
+      appPage.page.getByText(
+        /This claim has been verified, expires on January 25, 2030/i,
+      ),
+    ).toBeVisible();
+
+    // veteran verification
+    await appPage.page
+      .getByRole("button", { name: "Edit Veteran status" })
+      .click();
+    await appPage.page
+      .getByRole("radio", { name: "This claim does not apply to" })
+      .click();
+    await appPage.page.getByRole("button", { name: "Save changes" }).click();
+    await expect(
+      appPage.page.getByText(/This claim does not apply to/i),
+    ).toBeVisible();
+
+    // notes
+    await appPage.page.getByRole("button", { name: "Add notes" }).click();
+    await appPage.page
+      .getByRole("textbox", { name: "Notes" })
+      .fill("Notes notes notes");
+    await appPage.page.getByRole("button", { name: "Save changes" }).click();
+    await appPage.waitForGraphqlResponse("PoolCandidate_UpdateNotes");
+    await expect(
+      appPage.page.getByRole("button", { name: "Edit notes" }),
+    ).toBeVisible();
+    await expect(appPage.page.getByText(/Notes notes notes/i)).toBeVisible();
+  });
+
+  test("Completing an assessment step", async ({ appPage }) => {
+    await loginBySub(appPage.page, "admin@test.com");
+    await appPage.page.goto(`/en/admin/candidates/${candidate.id}/application`);
+    await appPage.waitForGraphqlResponse("PoolCandidateSnapshot");
+
+    // to assess icon by application screening
+    await expect(
+      appPage.page.getByLabel("To assess").locator("path"),
+    ).toBeVisible();
+
+    // education result
+    await appPage.page
+      .getByRole("row", { name: "Education requirement To" })
+      .getByRole("button")
+      .click();
+    await expect(
+      appPage.page.getByText("I meet the applied work"),
+    ).toBeVisible();
+    await expect(
+      appPage.page
+        .getByLabel("Assess the candidate's")
+        .getByText("Test Experience"),
+    ).toBeVisible();
+    await appPage.page.getByText("Demonstrated", { exact: true }).click();
+    await appPage.page
+      .getByText("Work experience equivalency is accepted", { exact: true })
+      .click();
+    await appPage.page.getByRole("button", { name: "Save decision" }).click();
+    await expect(
+      appPage.page
+        .getByLabel("Assess the candidate's")
+        .getByText("Test Experience"),
+    ).toBeVisible();
+    await expect(
+      appPage.page.getByRole("button", {
+        name: "Demonstrated",
+      }),
+    ).toBeVisible();
```

Comment:
These tests also uncovered something, not sure how great it is that all these buttons have the same text :cry: 

I'm going to do some testing but we may want to change these so the name is like "Education assessment, application screening demonstrated"

Reply:
In case you are wondering, I did do the testing and the issue was the table markup. 

https://github.com/GCTC-NTGC/gc-digital-talent/issues/12207